tests/consensus_tests/test_named_vector_crud.py

```
@@ -158,6 +158,140 @@ def test_create_vector_no_optimization(tmp_path: pathlib.Path):
+       optimization is in flight, delete the `drop` vector — overlapping the
+       deletion with an active rebuild. Force a couple more migrations to keep
+       the optimizer churning while the schema settles.
+    4. Assert the collection returns to green (optimizations progress, not
+       stuck), all points survive, search on `keep` still works, and `drop` is
+       gone.
+    """
+    VECTOR_DIM = 32
+    N_POINTS = 1000
+    assert_project_root()
+
+    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, N_PEERS)
```

Ruff (0.15.20)

[warning] Unpacked variable `peer_dirs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)